infra/dev-personal/main.tf

```diff
@@ -128,3 +128,53 @@ module "psoxy-google-chat" {
     }
   }
 }
+
+locals {
+  # set this to no if it's the first time creating the slack function
+  slack_secret_created  = "no"
+  slack_service_account = "psoxy-slack-discovery-api@psoxy-dev-jose.iam.gserviceaccount.com"
```

Comment:
@eschultink I need help this, I wasn't able to create the service account, so had to do it by hand in the console. I appreciate you taking a look to this.

Reply:
can you explain further? I see you as owner of your dev project, so it shouldn't be a permissions thing.

Perhaps an API enabling thing?

Reply:
hmm, not sure what happened back then. I just deleted the old one it created a new one... first time I had to create it manually. Maybe my environment was not properly setup.

Reply:
So now works, nevermind.